lib/assert/default_suite.rb

```diff
@@ -12,36 +12,56 @@ def initialize(config)
       reset_run_data
     end
 
-    def test_count;          @test_count;          end
-    def result_count;        @result_count;        end
-    def pass_result_count;   @pass_result_count;   end
-    def fail_result_count;   @fail_result_count;   end
-    def error_result_count;  @error_result_count;  end
-    def skip_result_count;   @skip_result_count;   end
-    def ignore_result_count; @ignore_result_count; end
+    attr_reader :test_count
+
+    attr_reader :result_count
+
+    attr_reader :pass_result_count
+
+    attr_reader :fail_result_count
+
+    attr_reader :error_result_count
+
+    attr_reader :skip_result_count
+
```

Comment:
@kellyredding - Do you want an empty line between all of these? Also not sure if they should be at the top of the file if they are `attr_reader`s.

Reply:
yeah - this was auto correct doing its thing and I didn't catch it - thanks!